parsing/ast_invariants.mli

```diff
@@ -19,5 +19,5 @@
 
 *)
 
-val structure : Parsetree.structure -> unit
-val signature : Parsetree.signature -> unit
+val implementation : Parsetree.implementation -> unit
+val interface : Parsetree.interface -> unit
```

Comment:
Introducing the new functions is fine, but I would be in favor of keeping the old ones for backwards compatibility.

Reply:
I agree that we should not remove function working on `structure`s or `signature`s.